nautobot_ssot/integrations/librenms/constants.py

```diff
@@ -379,3 +384,25 @@
     "ZTE": ["zxdsl"],
     "Zyxel": ["zynos", "zywall", "zyxelnwa", "zyxelwlc"],
 }
+
+# LibreNMS | Normalized
+LIBRENMS_LIB_MAPPER = {
```

Comment:
These should be moved to netutils if there isn't already a mapping there that would work for this.

Reply:
They're only here to get this completed without needing an update to Netutils and to make sure values were working.

The plan was that after this was working as expected, those would be moved to netutils and this updated to use it.